prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -464,6 +464,15 @@ def prompt_processor() -> Any:
             if assertion_failed or answer.lower() == "na":
                 structured_output[output[PSKeys.NAME]] = None
             else:
+                prompt = f'Extract yes/no from the following text:\n{answer}\n\n\
+                    Output in single word.\
+                    If the context is trying to convey that the answer is true, \
+                    then return "yes", else return "no".'
```

Comment:
@tahierhussain NIT : For more accurate prompts, we can explicitly mention explanation is not needed. [Ref](https://github.com/Zipstack/unstract/blob/7d9b6d0b7506a9f276080a7995229ceada8ad137/prompt-service/src/unstract/prompt_service/main.py#L420)